omnisafe/adapter/offpolicy_adapter.py

```diff
@@ -126,9 +126,10 @@ def rollout(  # pylint: disable=too-many-locals
         """
         for _ in range(rollout_step):
             if use_rand_action:
-                act = torch.as_tensor(self._env.sample_action(), dtype=torch.float32).to(
-                    self._device,
-                )
+                act = torch.normal(
```

Comment:
Why?

Reply:
This is because the action sampled from the environment already conforms to the action scale. Placing this action in the `step` function will result in it being scaled again by the `ActionScale` wrapper. Therefore, it is more reasonable to directly use Gaussian distribution sampling here.

Reply:
Doesn't rescaling the values already within the range keep them unchanged? A 0,1 normal distribution will lead to sampling results significantly biased towards [-1, 1]*scale, especially around 0. I wonder if this has a positive impact on exploration.

Reply:
Thank you for your reminder. I believe using a uniform distribution between -1 and 1 will be more advantageous for sampling.